theurgy.lic

```diff
@@ -62,6 +62,69 @@ def clean_altar
   put_in_container($settings.water_holder)
 end
 
+def prayer_bead
+  return unless DRSkill.getrank('Mechanical Lore') >= 140
+
+  fput "stow #{GameObj.right_hand.noun}" if GameObj.right_hand.name != 'Empty'
+  fput "stow #{GameObj.left_hand.noun}" if GameObj.left_hand.name != 'Empty'
+  get_from_container('prayer chain')
+  chain_contents = DRC.bput("look at my chain", /^Strung on to the prayer bead chain you see .*\./, 'There are currently no beads on it.')
+  bead_count = DRC.list_to_nouns(chain_contents.match(/^Strung on to the prayer bead chain you see (.*)\./).to_a[1])
+  return unless bead_count.count < 10 
+  put_in_container('prayer chain')
+  DRC.forage? 'limb'
+  get_from_container($settings.water_holder)
+  fput("sprinkle #{$settings.water_holder} on my limb")
+  put_in_container($settings.water_holder)
+  if 'interrupt your research' == DRC.bput('prep bless', 'Bless spell', 'interrupt your research')
+    find_project = fput("research status")
+    if find_project.include? "Fundamental"
+      hold_research = 'FUNDAMENTAL'
+    end
+    if find_project.include? "Augmentation"
+      hold_research = 'AUGMENTATION'
+    end
+    if find_project.include? "Stream"
+      hold_research = 'STREAM'
+    end
+    if find_project.include? "Sorcery"
+      hold_research = 'SORCERY'
+    end
+    if find_project.include? "Utility"
+      hold_research = 'UTILITY'
+    end
+    if find_project.include? "Warding"
+      hold_research =  'WARDING'
+    end
+    fput 'prep bless'
+  end
+  pause 5
+  fput 'cast my limb'
+  if hold_research
+    fput ("RESEARCH #{hold_research} 300")
+  end
+  get_from_container('carving knife')
+  while checkright == 'limb'
+    fput('carve my limb with my knife')
+    pause 1
+    waitrt?
+  end
+  put_in_container('carving knife')
+  get_from_container('shaper')
+  while checkright == 'block'
+    fput("shape block to #{$settings.aspect}")
```

Comment:
yeah - I just haven't pulled the base.yaml for it yet.  It would be otherwise I would have to run a check against your favored god and hope that your last favor was from that god.  This way you have more control.